lib/matplotlib/__init__.py

```diff
@@ -749,19 +744,31 @@ def _open_file_or_url(fname):
             yield f
 
 
-def _rc_params_in_file(fname, fail_on_error=False):
+def _rc_params_in_file(fname, transform=lambda x: x, fail_on_error=False):
```

Comment:
Can you put transfrom last for API change nit-picks?

Reply:
this is private, and logically transform comes first...

Reply:
fair RE it being private.